client/app/api.js

```diff
@@ -1,150 +1,223 @@
     myStatus: 'declined',
     showtime: '2017-04-26T22:30:00.000Z',
     location: {
-      name: 'AMC Loews Boston Common 19',
-      lat: '42.353290',
-      long: '-71.061698',
-      address: '175 Tremont St, Boston, MA 02111, USA',
+        name: 'AMC Loews Boston Common 19',
+        lat: '42.353290',
+        long: '-71.061698',
+        address: '175 Tremont St, Boston, MA 02111, USA',
     },
     runtime: '129',
     rating: 0.71,
     thumbnail: 'https://image.tmdb.org/t/p/original/8LdkswnaKrZbySdxdkoZCB5QDto.jpg',
     backdrop: 'https://image.tmdb.org/t/p/original/6aUWe0GSl69wMTSWWexsorMIvwU.jpg',
     organizer: {
-      name: 'Peter Darrow',
+        name: 'Peter Darrow',
     },
     notes: "I'm really looking forward to seeing this! Let's grab a bite to eat afterward.",
-    invitees: [
-      {
+    invitees: [{
         name: 'Summers Pittman',
         status: 'accepted',
-      },
-      {
+    }, {
         name: 'Emilio Rodriguez',
         status: 'declined',
-      },
-      {
+    }, {
         name: 'Damien Murphy',
         status: 'unanswered',
-      },
-    ],
-  },
-  {
+    }, ],
+}, {
     title: 'The Boss Baby',
     synopsis: "A story about how a new baby's arrival impacts a family, told from the point of view of a delightfully unreliable narrator, a wildly imaginative 7 year old named Tim.",
     myStatus: 'unanswered',
     showtime: '2017-04-29T23:45:00.000Z',
     location: {
-      name: 'Landmark Kendall Square',
-      lat: '42.367819',
-      long: '-71.090022',
-      address: 'One Kendall Square, 355 Binney St, Cambridge, MA 02139, USA',
+        name: 'Landmark Kendall Square',
+        lat: '42.367819',
+        long: '-71.090022',
+        address: 'One Kendall Square, 355 Binney St, Cambridge, MA 02139, USA',
     },
     runtime: '97',
     rating: 0.53,
     thumbnail: 'https://image.tmdb.org/t/p/original/vkvPL8Lef9grxx6tyrojrjaaAw4.jpg',
     backdrop: 'https://image.tmdb.org/t/p/original/bTFeSwh07oX99ofpDI4O2WkiFJ.jpg',
     organizer: {
-      name: 'Peter Darrow',
+        name: 'Peter Darrow',
     },
     notes: "I'm really looking forward to seeing this! Let's grab a bite to eat afterward.",
-    invitees: [
-      {
+    invitees: [{
         name: 'Summers Pittman',
         status: 'accepted',
-      },
-      {
+    }, {
         name: 'Emilio Rodriguez',
         status: 'declined',
-      },
-      {
+    }, {
         name: 'Damien Murphy',
         status: 'unanswered',
-      },
-    ],
-  },
-];
+    }, ],
+}, ];
+
+/**
+ * Create a new invite and notify invitees.
+ *
+ * @param {object} invite Invite to be created. Will look like one of the
+ * objects defined in `mockInviteData`.
+ * @returns {Promise} Does not need to return anything when resolved.
+ */
+function createInvite(invite) {
+    return HttpClient.post('invites', invite)
+        .then(invite => {
+            return invite;
+        })
+        .catch(err => {
```

Comment:
Does this prevent the error from being handled in the React components (like [https://github.com/feedhenry/snapscreen/blob/create-invite/client/app/screens/InviteList.js#L80])? I'm not super familiar with aysnc yet.